core/app/models/spree/refund_reason.rb

```diff
@@ -2,7 +2,10 @@
 
 module Spree
   class RefundReason < Spree::Base
-    include Spree::NamedType
+    scope :active, -> { where(active: true) }
+    default_scope -> { order(arel_table[:name].lower) }
```

Comment:
Is there any value in having this "metaprogramming" now?

Reply:
Yes, default scopes are troublemakers and eventually we'd like to migrate them to regular scopes (e.g. `scope :order_by_name …`) so they don't get in the way with other things (joins, batch loading, …). While checking where the default scopes were defined we thought that this "concern" was not actually shared behavior but would be better off as local model features. 

I hope I got the question right, otherwise let me know 🙏

Reply:
@kennyadsl see also https://github.com/solidusio/solidus/pull/5539/commits/b2d4a7cdc73c7aae2bf5da0ee357cdf4944b031b